package/src/contexts/messageInputContext/MessageInputContext.tsx

```diff
@@ -338,6 +339,11 @@ export type InputMessageInputContextValue<
    */
   editing?: MessageType<StreamChatGenerics>;
 
+  /**
+   * Prop to override the emojis in auto complete suggestion list.
+   */
+  emojis?: CompiledEmojis;
```

Comment:
can we align this type to react as much as we can.. currently still looks like the previous RN type

Reply:
Done